metaflow/plugins/kfp/kfp_exit_handler.py

```diff
@@ -88,6 +147,25 @@ def email_notify(send_to):
     else:
         print("No notification is necessary!")
 
+    # Send message to SQS if 'METAFLOW_SQS_URL_ON_ERROR' is set
+    metaflow_sqs_url_on_error = get_env("METAFLOW_SQS_URL_ON_ERROR")
+
+    if metaflow_sqs_url_on_error:
+        if status == "Failed":
+            message_body = sqs_message_json
+            print(f"message to be sent: {message_body}")
```

Comment:
could this have PII data that we shouldn't log to STDOUT and Splunk?

Reply:
good catch. Forgot to remove it after debugging. It is removed now.